nbqa/config/config.py

```diff
@@ -208,3 +246,28 @@ def validate(self) -> None:
                     """
                 )
             )
+        if self.nbqa_config and not Path(self.nbqa_config).exists():
+            raise FileNotFoundError(f"{self.nbqa_config} not found.")
+        if (
+            self.nbqa_config
```

Comment:
This condition can be simplified to 

```Python
if command in CONFIG_FILES.keys() and self.nbqa_config not in CONFIG_FILES[command]
```

instead of checking self.nbqa_config, we can check once at the top. Also this validate method could be split in to smaller methods

```Python
def validate(self, command):
      _validate_nbqa_args()
      _validate_config_file_exists()
      _validate_config_file_for_command()
```

Reply:
```
if command in CONFIG_FILES.keys() 
```
problem is, this is a default dict, so just accessing an element adds it to the keys.

e.g.:
```
>>> from collections import defaultdict
>>> d = defaultdict(list)
>>> d.keys()
dict_keys([])
>>> d[9]
[]
>>> d.keys()
dict_keys([9])
```

Reply:
But `command in config_files. keys()` would not add it to the dictionary, because we are not checking directly on the dict rather on the result returned by the keys method

Reply:
I am in favor of removing this check.